aws/src/main/java/org/frankframework/filesystem/AmazonS3FileSystem.java

```diff
@@ -267,6 +267,11 @@ public boolean exists(S3Object f) {
 		return s3Client.doesObjectExist(bucketName, f.getKey());
 	}
 
+	@Override
+	public boolean isFolder(S3Object s3Object) {
+		return s3Object.getKey().endsWith("/");
```

Comment:
Unfortunately when testing against a real S3, it turns out that this check does not work and it fails the test `FileSystemSenderTest#fileSystemSenderListFilesAndFoldersActionTestInFolder`.

Too bad, we need to rework now the solution we thought of, potentially costing us performance.

Reply:
Yes, because it's a flat-file structure, there are no folders in AWS...

Reply:
Yeah, I just read: https://docs.aws.amazon.com/AmazonS3/latest/userguide/using-folders.html
A 'folder' is actually a '0-byte object with a key that's set to the folder name'. 
Shall we disallow the listing of folders in S3?

Reply:
As just discussed with Niels, I am going to check with S3 myself how to best fix this.

Reply:
It turns out the problem was with the test.